src/utils.js

```diff
@@ -187,10 +187,10 @@ var Utils = exports.Utils = {
    * @returns {Boolean} true if disabled
    **/
   isPrivacyBadgerEnabled: function(origin){
-    if(localStorage.disabledSites && JSON.parse(localStorage.disabledSites).length > 0){
+    if(localStorage.disabledSites){
       var sites = JSON.parse(localStorage.disabledSites);
-      for(var i = 0; i < sites.length; i++){
-        if(sites[i] === origin){ return false; }
+      if(sites[origin]){
+        return false;
```

Comment:
This is a good start, but it still causes a disk read every time we call `isPrivacyBadgerEnabled`. We can get even more of a speed gain by reading localStorage.disabledSites once at startup and storing that in an object in the global scope for utils.js. This would also give us an elegant solution to the issue below.